dipy/viz/horizon/util.py

```diff
@@ -12,3 +23,42 @@ def check_img_shapes(images):
         if base_shape != data.shape[:3]:
             return False
     return True
+
+
+def check_img_dtype(images):
+    """Check for supported dtype. If not supported numerical type, fallback to
```

Comment:
after the first `.` you should return to a new line. see [here](https://github.com/dipy/dipy/blob/master/dipy/direction/peaks.py#L94-L131) for an example